### DISCUSSION TITLE
What is the average response time for critical downtime?

### DISCUSSION THREAD

**Discussion by JazzBashara** [vercel/community#5808] [Account Issue] [answered]
### Question

I opened 2 tickets but haven't had a response Severity 1 (Active production downtime, no workaround).

Is there outside Vercel entreprise any other possible way to get prio support? Dealing with crucial down time and not being able to get a reply within a couple hours has been hard so far. Hope someone can pick any of these tickets up!

Appreciate it, thanks.

#00197080 -  #00197229

@swarnava @amyegan

**Comment by amyegan** [comment, marked as answer]
Hi @JazzBashara. It looks like your case has a response and your domain was renewed. Please follow up with case 00197290 if you have any other questions 🙂
